### ISSUE TITLE
[SUGGESTION] WZS: Improve map voting

### ISSUE THREAD

**Issue by CWorm** [Azurblau/Zombies.Zone#101]
Too many people follow other votes just beacuse they have no clue what to vote on, and bad maps get voted too often, [three servers, one map](https://media.discordapp.net/attachments/288761962240671746/430445199743909900/unknown.png)

A: Give votepower lv/50, let the ZZ community vote, not random kleiners for the map
B: Give votepower depending on best awards on end round (minus achievements like killing crows or dying close to spawn)
C: Hide votes from players so everyone votes for what they want to play

**Comment by Kowoalski** [comment]
Would it be possible to reduce the amount of options during map voting, and make it display maps that hadn't been picked in a while, say 7-10 matches? Otherwise people will pick mlt_defense or mall just because they're already used ot them, without trying out any of the underplayed maps. And yes, I do realise some maps aren't played often because they are bad/hard to survive on, but the reason they are that way can't be truly pinned down without them actually being played more often.

**Comment by CWorm** [comment]
> [...]but the reason they are that way can't be truly pinned down without them actually being played more often.

I agree with that, this is why I want to improve map voting as well, due to fact we can modify maps with lua it also means we can attempt to tweak them just like I did with Scrapmetal and Swimming Pool

**Comment by WardenPotato** [comment]
I think A is the best option since going support would mean you have less of a vote which i dont like.

**Comment by TheFray89** [comment]
i totally agree with B. 
People who have did a great match deserve to have more votepower than others

**Comment by v1red** [comment]
Or D every map to have an image as it is on killa server(gun game zombie mod or something like that) seeing a map will be easier for new players. And mabye less maps on vote mabye 10 and rotate them. Lets say 9 zs maps plus 1 obj. 
A. Level 50 hmmm a bit to mutch an unfair everyone should have the right to vote my opinion and what will happen if they are only newcommers online?
B. This will make like percent votting or extra votes for achivments or what will it be???
C. Totaly agains this.

**Comment by CWorm** [comment]
@v1red - I meant that lv 1-49 have 1 vote power, 50-99 have 2 vote power, 100-149 have 3 vote power and so on, they gonna have votes but those who played longer would have greater votepower

Regarding B, achievements on end round means the awards for most healing done, most repairs done, most damage to humans etc.
